scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -31,8 +34,11 @@ remove_dotnet_pkgs
 [ "$?" -ne 0 ] && echo "Failed to remove dotnet packages." && exit 1
 
 echo "Deleting install root - $dotnet_install_root"
-rm -rf $dotnet_install_location
+rm -rf $dotnet_install_root
 rm -f $dotnet_path_file
 
+echo "Deleting NuGet package location"
```

Comment:
Actually there are multiple nuget cache folders you may have to delete
•	~/.local/share/NuGet/v3-cache
•	~/.nuget/packages
•	/tmp/NuGetScratch/

cc: @joelverhagen @emgarten